test/test_ngram.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import os
+import kenlm
```

Comment:
If you think even kenlm python binding is difficult to install by default, you can skip this test by `kenlm = pytest.importorskip("kenlm")`  https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency